rules/do-you-implement-DevOps-practices-for-continuous-delivery/rule.md

```diff
@@ -1,14 +1,15 @@
 ---
 type: rule
 title: Do you implement DevOps practices for continuous delivery?
-uri: do-you-implement-DevOps-practices-for-continuous-delivery
+uri: implement-DevOps-practices-for-continuous-delivery
```

Comment:
As per my other comments please update the folder to match the new uri

Reply:
Maybe doesn't make a practical difference, but please also make the URI and folder in all lowercase

Reply:
Done ✅